repos/spack_repo/builtin/packages/colima/package.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack_repo.builtin.build_systems.makefile import MakefilePackage
+
+from spack.package import *
+
+
+class Colima(MakefilePackage):
```

Comment:
```suggestion
from spack_repo.builtin.build_systems.go import GoPackage

from spack.package import *


class Colima(GoPackage):
```

I would highly recommend using `GoPackage` here instead to pick up the optimizations and isolation we do for go builds.